core/src/docs/concepts.rs

```diff
@@ -53,15 +61,20 @@
 //! ```
 //!
 //! # Operator
-//!
-//! The [`Operator`] is a bridge between the underlying implementation details and the unified abstraction. OpenDAL will erase all generic types and higher abstraction around it.
+//! The [`Operator`] is a delegate for underlying implementation detail, and provides one unified access interface.
+//! It will hold one reference of Service with its all generic types erased by OpenDAL, which is the reason why we
+//! say the Operator is the delegate of one Service.
+//! And the Service also has another name called [`Accessor`][crate::raw::Accessor].
 //!
 //! ```text
-//! ┌───────────┐           ┌───────────┐              ┌─────────────────┐
-//! │           │  build()  │           │  type erase  │                 │
-//! │  Builder  ├──────────►│  Service  ├─────────────►│     Operator    │
-//! │           │           │           │              │                 │
-//! └───────────┘           └───────────┘              └─────────────────┘
+//!                          ┌─────────────────────────────┐
```

Comment:
`Operator` doesn't accept `Builder`, `OperatorBuilder` does. But we don't want to expose the `OperatorBuilder` in concept docs here.